webpack/components/AnsibleRolesSwitcher/AnsibleRolesSwitcherConstants.js

```diff
@@ -5,3 +5,8 @@ export const ANSIBLE_ROLES_ADD = 'ANSIBLE_ROLES_ADD';
 export const ANSIBLE_ROLES_REMOVE = 'ANSIBLE_ROLES_REMOVE';
 export const ANSIBLE_ROLES_ASSIGNED_PAGE_CHANGE =
   'ANSIBLE_ROLES_ASSIGNED_PAGE_CHANGE';
+export const ANSIBLE_ROLES_FORM_OBJECT = 'ANSIBLE_ROLES_FORM_OBJECT';
+export const ANSIBLE_VARIABLES_REQUEST = 'ANSIBLE_VARIABLES_REQUEST';
```

Comment:
There has been a change merged in core recently in API fetches.
see https://github.com/theforeman/foreman/pull/6739 or core storybook `APIMiddleware` documentation on how to use it.